platforms/xamarin.ios/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -0,0 +1,32 @@
+namespace System.Runtime.InteropServices
```

Comment:
Sorry that comment is a little out of context now because the baseline file had been updated since then :).